common/io/SelectServerTest.cpp

```diff
@@ -582,7 +582,7 @@ void SelectServerTest::testTimeout() {
       10,
       ola::NewSingleCallback(this, &SelectServerTest::ReentrantTimeout, m_ss));
   m_ss->RegisterSingleTimeout(
-      20,
+      40,
```

Comment:
Nope, just an old XP box. I think @lukase or someone mentioned that has a lower timer resolution than later OSes.